backend/siarnaq/api/compete/views.py

```diff
@@ -315,6 +316,59 @@ def scrimmage(self, request, pk=None, *, episode_id):
         serializer = self.get_serializer(page, many=True)
         return self.get_paginated_response(serializer.data)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="team_id",
+                type=int,
+                description="A team to filter for. Defaults to your own team.",
+            ),
+        ],
+        responses={
+            status.HTTP_204_NO_CONTENT: "No ranked matches found.",
+            status.HTTP_200_OK: HistoricalRatingSerializer(many=True),
+        },
+    )
+    @action(
+        detail=False,
+        methods=["get"],
+        permission_classes=(IsEpisodeMutable,),
+    )
+    def historical_ranking(self, request, pk=None, *, episode_id):
+        """List the historical rating of a team."""
+        queryset = Match.objects.all().filter(tournament_round__isnull=True)
+
+        team_id = parse_int(self.request.query_params.get("team_id"))
+        if team_id is not None:
+            queryset = queryset.filter(participants__team=team_id)
+        else:
+            queryset = queryset.filter(participants__team__members=request.user.pk)
+        has_invisible = self.get_queryset().filter(
```

Comment:
should we show invisible teams to staff, like in the scrimmage endpoint?

Reply:
IMO no need to bother with showing them to staff, for now?

Invisible teams don't really have a _good_ notion of rating anyways (since regular people can't really scrimmage them, they don't go into autoscrims, etc). Not worth the extra effort to put the code in. (unless there's a usecase I'm missing)